test/unit/mdc-textfield/foundation.test.js

```diff
@@ -126,6 +141,34 @@ test('#destroy deregisters blur handler', (t) => {
   t.end();
 });
 
+test('#destroy deregisters input handler', (t) => {
+  const {foundation, mockAdapter} = setupTest();
+  foundation.destroy();
+  t.doesNotThrow(() => td.verify(mockAdapter.deregisterInputInputHandler(td.matchers.isA(Function))));
+  t.end();
+});
+
+test('#destroy deregisters keydown handler', (t) => {
+  const {foundation, mockAdapter} = setupTest();
+  foundation.destroy();
+  t.doesNotThrow(() => td.verify(mockAdapter.deregisterInputKeydownHandler(td.matchers.isA(Function))));
+  t.end();
+});
+
+test('#destroyAutoCompleteHandlers deregisters input handler', (t) => {
```

Comment:
nit: The description here and below should be `#destroy deregisters ...`. We don't care about `#destroyAutoCompleteHandlers` since that's a private method and thus an implementation detail.